pkgs/tools/networking/networkmanager/openvpn/default.nix

```diff
@@ -21,11 +21,11 @@
 
 stdenv.mkDerivation rec {
   pname = "NetworkManager-openvpn";
-  version = "1.10.2";
```

Comment:
This requires NM 1.48.2. https://github.com/NixOS/nixpkgs/pull/321585#issuecomment-2183462987

Reply:
And while NetworkManager updates can go to master nowadays, stable is on NM 1.46 so it needs a separate update to 1.46.2.

Reply:
Does NM 1.46.2 supports networkmanager-openvpn 1.12.0?

We can do a partial backport of this to stable then.